ivy_tests/test_ivy/test_frontends/test_tensorflow/test_signal.py

```diff
@@ -10,35 +9,45 @@
 
 # hann_window
 
-
-@handle_cmd_line_args
-@given(
+handle_cmd_line_args
```

Comment:
actually, if you look at other current tests like in ivy/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_activations.py, there should be an "@" in front of the handle_cmd_line_args and given. The example given in the deep dive doc is wrong as the doc generating script we are using ignored "@"s. Thank you.